darshan-util/pydarshan/darshan/tests/test_summary.py

```diff
@@ -118,6 +118,10 @@ def test_main_without_args(tmpdir, argv, expected_img_count):
                     # check that expected number of figures are found
                     assert report_str.count("img") == expected_img_count
 
+                    # check the number of opening section tags
+                    # matches the number of closing section tags
+                    assert report_str.count("<section>") == report_str.count("</section>")
```

Comment:
Maybe add this to `test_main_all_logs_repo_files()` below as well, after rebasing to deal with current merge conflicts.

Since this test now covers at least one log repo file, the need for `test_main_all_logs_repo_files()` may become a point of contention, but for now, that test still remains useful for PRs to the logs repo, since it will scrape in every log to sanity check report generation.

Reply:
> Maybe add this to `test_main_all_logs_repo_files()` below as well, after rebasing to deal with current merge conflicts.

Thanks, I've rebased and added the same check to `test_main_all_logs_repo_files()`. 

> the need for `test_main_all_logs_repo_files()` may become a point of contention

We could probably remove it and move some test cases to `test_main_without_args` if we think they are important to keep. Then add write a new test that initializes a `DarshanReport` for each log in the repo and maybe perform a few checks on the log data.

Reply:
For now, making sure that every log file in the log repo can successfully be used generate a report file proper makes sense to me.